workflow-service/src/main/java/com/redhat/parodos/project/dto/response/ProjectAccessRequestDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.parodos.project.dto.response;
+
+import java.util.UUID;
+
+import com.redhat.parodos.project.enums.ProjectAccessStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ProjectAccessRequestDTO {
+
+	private UUID id;
+
+	private UUID projectId;
+
+	private String role;
+
+	private UUID userId;
```

Comment:
Isn't preferable to return username instead?

Reply:
username might be duplicate,  but if @wKich prefers username to facilitate UI display, I can change it

Reply:
@RichardW98 we need to show `firstName` and `lastName` in UI, because uuid doesn't say anything to end users, like they simply will not understand it :)